compiler-core/src/language_server/code_action.rs

```diff
@@ -3247,3 +3252,487 @@ impl<'a> DecoderPrinter<'a> {
         )
     }
 }
+
+/// Builder for code action to destructure a function argument:
+///
+/// ```gleam
+/// pub fn wibble(arg: Wibble) {
+/// //             ^ Cursor here
+/// }
+///
+/// // Adds this to the top of the function:
+///
+/// pub fn wibble(arg: Wibble) {
+///   let Wibble(label1:, label2:) = arg
+/// }
+/// ```
+///
+pub struct PatternMatchOnArgument<'a, A> {
```

Comment:
I think the name and documentation for this code action may be out of date now. It works for both function arguments and assignments?

Reply:
Oops! Fixed it

Reply:
The name is still out of date and the documentation is now missing!

Reply:
I really messed it didn't I 😆 It should be ok for real this time